plugins/services/src/js/utils/MarathonErrorUtil.js

```diff
@@ -0,0 +1,121 @@
+import ValidatorUtil from '../../../../../src/js/utils/ValidatorUtil';
+
+const MarathonErrorUtil = {
+
+  /**
+   * Try to guess an error type from it's contents
+   *
+   * @param {String} message - The marathon error string
+   * @returns {String} Returns the message type
+   */
+  /* eslint-disable no-unused-vars */
+  getErrorType(message) {
+  /* eslint-enable no-unused-vars */
+
+    // TODO: This will be implemented on the next PR
+    return 'MARATHON_ERROR';
+
+  },
+
+  /**
+   * Parses errros from the given input format into the common error format
+   * used internally
+   *
+   * For reference, marathon responds an error according to the exception that
+   * occurred and it's translated with the following checks:
+   *
+   * https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/api/MarathonExceptionMapper.scala#L63
+   *
+   * @param {Object|String} error - The marathon error(s) to process
+   * @returns {Array} Returns an array of error objects
+   */
+  parseErrors(error) {
+    if (error == null) {
+      return [];
+    }
+
+    // `error` could be a string, if the error has no details
+    if (typeof error === 'string') {
+      return [
+        {
+          path: [],
+          message: error,
+          type: MarathonErrorUtil.getErrorType(error),
```

Comment:
Hey @mlunoe , I would vote against this principle in all of our static utilities. These are not methods, but static functions and `this` should be considered "undefined".